src/WebJobs.Script/Description/FunctionDescriptorProvider.cs

```diff
@@ -18,6 +18,7 @@ namespace Microsoft.Azure.WebJobs.Script.Description
     public abstract class FunctionDescriptorProvider
     {
         internal const string DefaultInputParameterName = "input";
+        internal const string DefaultPowershellInputParameterName = "inputdata";
```

Comment:
I'm not sure, but don't the design guidelines for PowerShell use camel case for variable names? https://msdn.microsoft.com/en-us/library/dd878270(v=vs.85).aspx. If so, this should be **inputData**